File: cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float change;
    do
    {
        change = get_float("Change owed: ");
    }
    while (change < 0);
    int c = round(change * 100);
    int q = c / 25;
    int d = c % 25 / 10;
    int n = c % 25 % 10 / 5;
    int p = c % 25 % 10 % 5;
    printf("%i\n", q + d + n + p);
}

File: cash.py
from cs50 import get_float
import math

while True:
    change = get_float("Changed owed: ")
    if change > 0:
        break

c = change * 100
q = math.floor(c / 25)
d = math.floor(c % 25 / 10)
n = math.floor(c % 25 % 10 / 5)
p = math.floor(c % 25 % 10 % 5)
print(q + d + n + p)